enterprise/cmd/frontend/internal/codeintel/resolvers/resolver.go

```diff
@@ -109,7 +124,7 @@ func (r *resolver) IndexConfiguration(ctx context.Context, repositoryID int) ([]
 	}
 
 	var indented bytes.Buffer
-	json.Indent(&indented, marshaled, "", "\t")
+	_ = json.Indent(&indented, marshaled, "", "\t")
 	return indented.Bytes(), nil
```

Comment:
```suggestion
	return indented.Bytes(), json.Indent(&indented, marshaled, "", "\t")
```

Add if you want, im impartial :slightly_smiling_face:

Reply:
I actually want to do this all the time...

Reply:
ikr?